ui/main/src/app/services/navigation/NavigationService.ts

```
@@ -0,0 +1,134 @@
+        // and after if user reload again, this time it is not encoded anymore by the browser
+        // So it ends up with 3 possible links: a none encoded link, an encoded link or a twice encoding link
+        // and we have no way to know which one it is when processing the url
+        //
+        // To solve the problem we encode two times the url before giving it to the browser
+        // so we always have a unique case : a double encoded url
+        let newUrl = urlSplit[0] + '#/businessconfigparty/' + encodeURIComponent(encodeURIComponent(menuId)) + '/';
+
+        if (urlExtension) newUrl += encodeURIComponent(encodeURIComponent(urlExtension));
+        document.location.href = newUrl;
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)